src/app/components/summary-actionbar/index.html

```diff
@@ -84,6 +84,7 @@
   <stache-row>
     <stache-column screenSmall="6">
 			<p>Actions that can be taken on multiple items in a list of items.</p>
+			<p>When used with a multiselect list, hide the summary action bar until the user selects an item in the list. Once an item is selected the summary action bar should fly in from the bottom of the page.</p>
```

Comment:
I'd change "the user selects" to "users select." And I'd change "Once" to "After" (it's a translation/localization thing). Also, we should have a comma after the opening clause in the second sentence, but I'd also reword the opening clause to use active voice: "After users select an item, the summary..." And should "should fly in" be "flies in"? I mean, is there an alternative to flying in from the bottom or the page?

Reply:
Right now the flyin is not built into the component. Not sure what our standards are around wording things that are part of a pattern but need to be manually implemented.

Reply:
OK. Thanks.